src/nni_manager/training_service/kubernetes/kubernetesTrainingService.ts

```diff
@@ -327,5 +329,34 @@ abstract class KubernetesTrainingService {
 
         return Promise.resolve();
     }
+
+    protected async createRegistrySecret(filePath: string | undefined): Promise<string> {
+        if(filePath === undefined || filePath === '') {
+            return '';
```

Comment:
What does it mean to the caller if you return ''?

Reply:
fixed, changed to `undefined`